db/migrate/20160223125636_change_submission_migrates.rb

```diff
@@ -0,0 +1,8 @@
+class ChangeSubmissionMigrates < ActiveRecord::Migration
```

Comment:
Proszę to usunąć - zmiana miała być tylko w pliku `schema.rb` - u pozostałych osób ten plik jest ok.